src/frag.js

```diff
@@ -176,6 +176,10 @@ function insertBefore(
 	insertNode,
 	insertBeforeNode,
 ) {
+	if (isFrag(this) && insertNode.parentElement) {
+		return insertNode;
+	}
+
```

Comment:
I think this logic could be improved since it's possible `insertNode` is a fragment. Will followup in a different PR.